code/modules/hydroponics/seed_datums.dm

```diff
@@ -1332,4 +1341,50 @@
 	set_trait(TRAIT_PRODUCT_ICON,"bulb")
 	set_trait(TRAIT_PRODUCT_COLOUR,"#ffeedd")
 	set_trait(TRAIT_PLANT_ICON,"stalk")
-	set_trait(TRAIT_WATER_CONSUMPTION, 5)
\ No newline at end of file
+	set_trait(TRAIT_WATER_CONSUMPTION, 5)
+
+/datum/seed/mushroom/nfrihi
+	name = "nfrihi"
+	seed_name = "blizzard ears"
+	seed_noun = "nodes"
+	display_name = "blizzard ear stalks"
+	mutants = null
+	chems = list("flour" = list(10,10))
+	splat_type = /obj/effect/plant
+	kitchen_tag = "nfrihi"
+
+/datum/seed/mushroom/nfrihi/setup_traits()
+	..()
+	set_trait(TRAIT_MATURATION,6)
+	set_trait(TRAIT_PRODUCTION,1)
+	set_trait(TRAIT_YIELD,4)
+	set_trait(TRAIT_POTENCY,1)
+	set_trait(TRAIT_PRODUCT_ICON,"nfrihi")
+	set_trait(TRAIT_PRODUCT_COLOUR,"#DBDA72")
+	set_trait(TRAIT_PLANT_COLOUR,"#31331c")
+	set_trait(TRAIT_PLANT_ICON,"nfrihi")
+	set_trait(TRAIT_WATER_CONSUMPTION, 4)
+	set_trait(TRAIT_IDEAL_LIGHT, 3)
+	set_trait(TRAIT_IDEAL_HEAT, 253)
+/datum/seed/nmshaan
+	name = "nmshaan"
```

Comment:
Should use the translated name, like the other plants do.

Reply:
Nif Berries don't. Only thing that matters is that the display name and the seed packets are translated, which they are. These are just code identifiers.